rust/numaflow-core/src/pipeline/isb/jetstream/writer.rs

```diff
@@ -584,6 +584,25 @@ impl JetstreamWriter {
                                 ?ack,
                                 "Duplicate message detected"
                             );
+                            // Increment drop metric for duplicate messages
+                            pipeline_metrics()
+                                .forwarder
+                                .drop_total
+                                .get_or_create(&pipeline_drop_metric_labels(
+                                    this.vertex_type.as_str(),
+                                    stream.name,
+                                    "Duplicate ID",
```

Comment:
```suggestion
                                    "duplicate-id",
```

This is more cleaner?